ambry-utils/src/test/java/com.github.ambry.utils/SimpleByteBufferPoolTest.java

```diff
@@ -45,27 +48,30 @@ public void testSingleRequestAllocateDeallocate()
   }
 
   /**
-   * Scenario: requested size is larger than pool capacity.
+   * This scenario tests when a {@link SimpleByteBufferPool} tries to allocate a {@link ByteBuffer} that is larger
+   * than its capacity. This operation should result in an {@link IllegalArgumentException}.
    */
   @Test
   public void testRequestExceedPoolCapacity()
-      throws TimeoutException, InterruptedException {
+      throws Exception {
```

Comment:
I don't think there is any harm throwing an `Exception` here as the caller of this method does not care. There is no real value in stating what exceptions get thrown for these junit tests.